presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergHiveMetadata.java

```diff
@@ -268,6 +269,8 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
 
         Schema schema = toIcebergSchema(tableMetadata.getColumns());
 
+        verifyTypeSupported(schema);
```

Comment:
Can you please verify if we still need fix from addColumn() ? since beginCreateTable() will block it. 
```
if (columnType.equals(Types.TimestampType.withZone())) {
            throw new PrestoException(NOT_SUPPORTED, format("Iceberg column type %s is not supported", columnType));
        }
```

Reply:
As I understand, it's used for statement `alter table xx add column`;

Reply:
Changes looks good to me. Thanks !!